.circleci/config.yml

```diff
@@ -22,7 +22,8 @@ jobs:
 
     steps:
     - prometheus/setup_environment
-    - run: sudo apt-get -y install build-essential libsnmp-dev
+    - run: apt-get update
```

Comment:
Why no update any more?

Reply:
Change to the Prometheus golang-builder image. Doesn't include pre-populated apt package cache.